### ISSUE TITLE
No video, just a new picture every 15sec

### ISSUE THREAD

**Issue by consoleman** [KhaosT/homebridge-camera-ffmpeg#157]
Hey,

Just installed the plugin on my RPi with Jessie and Homebridge running.
I get no video in the Home App, just a updated picture every 15 seconds.
If I open the camera in Home app, there isn't any update, just in the "room screen"

Tried to change resolution, packet size and FPS but no change.

Any clues?

**Comment by sylarevan** [comment]
Hi there.
Same issue with the plugin used inside a working homebridge docker container. While I was able to get a video stream from my cameras before, it's now not working anymore. Can't exactly catch when it happened. So far getting a still image is OK. But nothing happens when trying to get the video stream up. I've tried some options mentioned here (packet size and rtsp_transport) with no success.

Importantly, the "bug" appears with my two cameras: a FOSCAM C2 and an old DLink webcam, both of them being "working" before (well, not so well, the video stream was sometimes corrupted... but at least I had a stream).
